home.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CC Global Summit</title>
+    <link rel="stylesheet" href="./home.css" />
+  </head>
+  <body>
+
+    <!-- Navigation Section Starts -->
+
+    <header>
+      <ul class="top-nav">
+        <li>
+          <img src="./images/facebook.png" alt="">
+        </li>
+        <li>
+          <img src="./images/twitter.png" alt="">
+        </li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+      <nav>
+        <div class="hamburger open" id="hamburger">&#9776;</div>
+        <a href="./home.html">
+          <img
+          class="nav-logo"
+          src="./images/logo.png"
+          alt="CC Global Summit Logo"
+          />
+        </a>
+        <ul class="nav-items">
+          <li class="nav-item"><a href="./about.html">About</a></li>
+          <li class="nav-item"><a href="">Program</a></li>
+          <li class="nav-item"><a href="">Join</a></li>
+          <li class="nav-item"><a href="">Sponsor</a></li>
+          <li class="nav-item"><a href="">News</a></li>
+          <li class="nav-item"><a href="" id="desktop-campaign">CC Campaign</a></li>
+        </ul>
+      </nav>
+
+      <!-- menu toggle -->
+
+      <div id="myNav" class="overlay close">
+        <a href="javascript:void(0)" class="closebtn close">&times;</a>
+        <div class="overlay-content">
+          <a href="./about.html" class="close">About</a>
+          <a href="#program" class="close">Program</a>
+          <a href="#join" class="close">Join</a>
+          <a href="#sponsor" class="close">Sponsor</a>
+          <a href="#news" class="close">News</a>
+          <a href="#cc-campaign" class="close" id="campaign">CC Campaign</a>
+        </div>
```

Comment:
- [ ] As this is a list, it will be better to put in a `ul` with the items included as list items using the `li` tag. Please kindly fix this.

Reply:
Done